filters/include/pcl/filters/filter_indices.h

```diff
@@ -109,7 +114,29 @@ namespace pcl
         deinitCompute ();
       }
 
-      /** \brief Set whether the regular conditions for points filtering should apply, or the inverted conditions.
+      /** \brief Calls the filtering method with specified executor and returns the filtered point cloud indices.
+        * \param[int] exec the executor to run the filter using
+        * \param[out] indices the resultant filtered point cloud indices
+        */
+      template <typename Executor>
+      void
+      filter (const Executor &exec, Indices &indices)
+      {
+        static_assert(pcl::is_invocable_v<
+                          decltype(&DerivedFilter::template applyFilter<Executor>),
+                          DerivedFilter&,
+                          Executor const&,
+                          PointCloud&>,
```

Comment:
Should be `Indices&`

Reply:
You should add the change in this PR nevertheless.